DGC.ValueSets.schema.json

```diff
@@ -6,105 +6,95 @@
   "$defs": {
     "disease-agent-targeted": {
       "description": "EU eHealthNetwork: Value Sets for Digital Green Certificates. version 1.0, 2021-04-16, section 2.1",
-      "type": "object",
-      "required": ["code", "system"],
-      "oneOf": [
-        {"code": { "const": "840539006"}, "system": { "const": "2.16.840.1.113883.6.96" }, "version": {"const": "2021-01-31"}}
-      ]
+      "type": "string",
+      "required": [ "code" ],
+      "enum": [ "840539006" ]
```

Comment:
As discussed.
The original had three fields: [code, system, version]. During review the decision was taken to reduce this to only "code" which is theoretically incorrect. However, since each of the values in our reduced value sets are unique then the decision was taken to drop system as it was redundant in ensuring uniqueness since the codes in and of themselves are unique in any case. This may change in the future though...

For now, am fine with collapsing this to the one field "code" but I left the "infra" of system/version in there in case we needed to expand again. That said, am fine with this edit and I guess we'll just have to expand again once we find a need for it.